internal/tiltfile/config/config.go

```diff
@@ -10,9 +10,9 @@ import (
 
 type Settings struct {
 	enabledResources []model.ManifestName
-	argDef           ArgsDef
+	configDef        ConfigDef
 
-	flagsParsed bool
+	configParsed bool
```

Comment:
i think this one SHOULD be `argsParsed`?

Reply:
or rather: would you call `configs.parse` even if no args were passed? Hmm, I guess maybe yes...

Reply:
In general I don't expect Tiltfile to make the `config.parse` call dependent on whether there are flags (and I don't think it's even non-trivial to write such a Tiltfile) - so, yes, with two sample use cases:
* you want to enable the "`tilt up` with no args starts all services", but with `config.parse` and `config.set_enabled_resources`. When there are no args, you still call flags.parse and get an empty array. (or maybe it's fairer to say `args` always has a value, but sometimes it's an empty array?)
* (not in master) you have a tilt_config.json and want the values from that

An explanation of this bool's purpose:
1. If a Tiltfile does not call `config.parse`, the args are used to select which resources to start.
2. If a Tiltfile does call `config.parse`, then we assume the Tiltfile does useful things with the args, and Tilt does not do anything with the args other than make sure they're an input to the handling of `config.parse` (with which the Tiltfile author might possibly then turn around and call `set_enabled_resources`)

Maybe `configParseCalled` would be clearer?